docs/_packages/oauth.md

```diff
@@ -288,6 +288,23 @@ const installer = new InstallProvider({
 ```
 ---
 
+### State verification
+
+By default, this package handles generating and verifying a `state` parameter during OAuth installation. This added measure helps to mitigate the risk of [Cross-Site Request Forgery](https://datatracker.ietf.org/doc/html/rfc6749#section-10.12) and is strongly recommended. 
+
+In specific installation scenarios, such as when an Org-Wide app is installed from an admin page, state verification cannot be completed because a `state` parameter isn't provided. In this case, you can disable `state` verification via setting the `InstallProvider#stateVerification` option to `false`. In this case, the installer will no longer require that `state` be present to proceed with installation. If `state` is provided in the request url parameters, it will still be verified.
```

Comment:
A few comments:

* How about being more specific here> In specific installation scenarios -> In specific installation scenarios with Enterprise Grid organizations?
* `In this case,` is repeated in two consecutive sentences. Can we change the transition words to something else?
* I think that we don't need to do the verification in this case because it's disabled> If `state` is provided in the request url parameters, it will still be verified.

Reply:
@seratch There's also another scenario, the Add to Slack button on the App config page also doesn't supply state. 
 
I'm thinking it's possible for an app to need to be installed from the`slack/install` page and from admin page / Add to Slack app configuration button. In that case a `state` param could sometimes be supplied and sometimes not. If an app is providing a state, wouldn't it be more secure to verify it?

Reply:
@srajiang For the scenario, our recommendation would be using Direct Install URL. As you know, we've recently added the support for it in bolt-js (it's not yet released though): https://github.com/slackapi/bolt-js/issues/977 With this way, you can use `/slack/install` for Add to Slack in App Directory.